spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -338,26 +338,17 @@ def get_ratings(app_id, platform, version_id = '', storefront = '')
       parse_response(r, 'data')
     end
 
-    def get_reviews(app_id, platform, storefront, version_id)
-      index = 0
+    def get_reviews(app_id, platform, storefront, version_id, page = 0, sort = 'REVIEW_SORT_ORDER_MOST_RECENT')
       per_page = 100 # apple default
```

Comment:
See https://github.com/BendingSpoons/bunker-api/pull/55#discussion_r191333860
It would be cool to either receive this as param or expose it as a constant